tests/app/test_info.py

```diff
@@ -0,0 +1,16 @@
+import unittest
+
+from frontstage import app
+
+
+class TestInfo(unittest.TestCase):
+
+    def setUp(self):
+        self.app = app.test_client()
+
+    def test_info_with_git_info(self):
+        response = self.app.get("/info")
+
+        self.assertEqual(response.status_code, 200)
+        self.assertTrue('"name": "ras-frontstage"'.encode() in response.data)
```

Comment:
Could you not use assert in rather than use assert true in?

Reply:
This makes a lot of sense, I think we use this assertTrue method in pretty much all the tests in frontstage/frontstage-api but I'll have a look at changing them